.changeset/hot-plums-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+"@scalar/fastify-api-reference": patch
+---
+
+Removed fastify-html to avoid conflicts
```

Comment:
this is incorrect. There are no conflicts.

Reply:
Yes, ofc! I'll change these messages. 

https://github.com/scalar/scalar/issues/1328#issuecomment-2025604485